### PR TITLE
Vendor

### PR DESCRIPTION
**Pull request by imakhlin**
* "vendorise" all dependencies
* Create `README.md` with revisions of the dependent packages

### CODE REVIEW

File: internal/pkg/performance/metrics.go [hunk 1]
```diff
@@ -20,11 +21,13 @@ var once sync.Once
 const (
 	STDOUT = "stdout"
 	STDERR = "stderr"
+
+	notRuningState = 0
+	runningState   = 1
```
Comment (gtopper) [inline internal/pkg/performance/metrics.go]: There's stuff in this PR that doesn't seem to be related to vendorization?

Comment (imakhlin) [reply internal/pkg/performance/metrics.go]: reverted this change

File: pkg/appender/appender.go [hunk 2]
```diff
@@ -38,8 +38,8 @@ const channelSize = 4048
 const maxSamplesBatchSize = 16
 const queueStallTime = 1 * time.Millisecond
 
-const minimalUnixTime = 20000000 // year 1970
-const maxUnixTime = 13569465600  // year 2400
+const minimalUnixTimeMs = 0          // year 1970
```
Comment (imakhlin) [inline pkg/appender/appender.go]: Note, changed minTime to `0` to allow integration tests to run as they should
